ratis-metrics/src/main/java/org/apache/ratis/metrics/impl/TimekeeperImpl.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.ratis.metrics.impl;
+
+import org.apache.ratis.metrics.Timekeeper;
+import org.apache.ratis.thirdparty.com.codahale.metrics.Timer;
+
+public class TimekeeperImpl implements Timekeeper {
```

Comment:
Whether there are other implementations of the Timekeeper interface in the future, if so, I suggest naming it as DefaultTimekeeperImpl

Reply:
That's  a good idea.  Let's call it `DefaultTimekeeperImpl`.